src/yetibot/commands/aws.clj

```diff
@@ -0,0 +1,107 @@
+(ns yetibot.commands.aws
+  (:require
+    [taoensso.timbre :refer [info]]
+    [yetibot.core.hooks :refer [cmd-hook]]
+    [yetibot.api.aws :as aws]))
+
+(defn iam-create-group-in-path-cmd
+  "aws iam create-group <path> <group-name> # Creates an aws IAM group named <group-name> within the specified <path>"
+  {:yb/cat #{:util :info}}
+  [{[_ path group-name] :match}]
+  (aws/iam-create-group path group-name))
```

Comment:
Hey @kaffein I had a thought – would be awesome if all the aws commands supported `data`, which means each command would return a structure like:

```clojure
{:result/value "formatted string value"
 ;; underlying data that was used to derive the formatted string
 :result/data ... }
```

Reply:
hey @devth 

thanks a lot for the feedback ;-)
I will refactor the commands to have the suggested _structure_ ...

:+1:

Reply:
when thinking about it @devth : maybe I should move all the output formatting logic upper in the stack (at the command level) ... I mean, I am supposed to implement only api calls here right ? though, I am mixing a lot of stuff instead.

what do you think ?

Reply:
I think you can do it either way. One case for formatters living outside of the command ns is if there are other commands or observers that could use the formatters. But I'm not sure we'd have that for aws. Another option if there are a lot of formatters would be to create a separate formatting util namespace for it. I'm fine with baking them into the command namespace too if you want.

Reply:
Alright @devth!!
Thanks a lot for the tips ;-) ...